src/ddmd/func.d

```diff
@@ -4213,11 +4213,15 @@ extern (C++) FuncDeclaration resolveFuncCall(Loc loc, Scope* sc, Dsymbol s,
         TypeFunction tf2 = m.nextf.type.toTypeFunction();
         const(char)* lastprms = parametersTypeToChars(tf1.parameters, tf1.varargs);
         const(char)* nextprms = parametersTypeToChars(tf2.parameters, tf2.varargs);
-        .error(loc, "%s.%s called with argument types %s matches both:\n%s:     %s%s\nand:\n%s:     %s%s",
+
+        const(char)* mod1 = MODtoChars(tf1.mod);
+        const(char)* mod2 = MODtoChars(tf2.mod);
+
+        .error(loc, "%s.%s called with argument types %s matches both:\n%s:     %s%s %s\nand:\n%s:     %s%s %s",
             s.parent.toPrettyChars(), s.ident.toChars(),
             fargsBuf.peekString(),
-            m.lastf.loc.toChars(), m.lastf.toPrettyChars(), lastprms,
-            m.nextf.loc.toChars(), m.nextf.toPrettyChars(), nextprms);
+            m.lastf.loc.toChars(), m.lastf.toPrettyChars(), lastprms, mod1,
+            m.nextf.loc.toChars(), m.nextf.toPrettyChars(), nextprms, mod2);
```

Comment:
@ZombineDev thanks! I added the first snippet of code that you suggested and it does the trick